### DISCUSSION TITLE
500 Error when Attempting to reach the Web UI

### DISCUSSION THREAD

**Discussion by knguyen298** [open-webui/open-webui#3800] [General]
# Bug Report

## Description

**Bug Summary:**
When trying to access Open-WebUI, a message shows up saying "500: Internal Error".

**Steps to Reproduce:**
Start up a fresh Docker container of both Open-WebUI and Ollama, and attempt to access it.

**Expected Behavior:**
The webpage loads.

**Actual Behavior:**
A message shows up displaying "500: Internal Error"

## Environment

- **Open WebUI Version:** v0.3.8-cuda
- **Ollama (if applicable):** v0.2.1

- **Operating System:** Ubuntu 20.04
- **Browser (if applicable):** Firefox 127.0.2

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
![image](https://github.com/open-webui/open-webui/assets/65993073/7983cc1e-d941-490d-a279-3c2e3128e0f2)

## Installation Method

Docker

## Additional Information

This bug started happening after the SSO fix was deployed. On v0.3.7, I was able to access the site fine, and could SSO into it using a Trusted Header. On the initial release of v0.3.8, SSO was broken, but I did not get error 500 if I disabled SSO and logged in normally. On this build of v0.3.8, error 500 will show up, both with SSO enabled and disabled.

Open-WebUI is being reverse proxied via Traefik2, with authentication being performed by Authelia. I did a full removal of Open-WebUI and Ollama yesterday, including deleting the docker images and data folders. After starting fresh new containers for both, the error message was immediately present.

I am currently using v0.3.7 without issue, and if I go to v0.3.8 without changing anything (open-webui:0.3.7-cuda -> open-webui:cuda), I still get an error message.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by knguyen298** [comment]
Unsure what I did, but it started working again.

**Comment by magnus919** [comment]
I get this error a lot but it doesn't show up in the logs. Logs are showing `200 OK` mostly. The only non-200 code I'm getting is `"GET /_app/version.json HTTP/1.1" 304 Not Modified`.

**Comment by 99rgosse** [comment]
I had the same issue, with fresh container & so on, so I assume some caching in Firefox would occur.
Launching localhost:3000/ollama showed up a 401 unauthorized, refreshed the page, then allowed to create again an admin account.

The issue is probably in the cached credentials somewhere.

**Reply by createchange** [reply]
Thank you!

**Comment by buchmeister** [comment]
When I got this error (running owu in Docker), I checked the docker logs and it was complaining about certificates, SSL routines, and hitting secured sites.
Now we have a squirrely, overly-locked down work network. But I launched the UI as **https://localhost:3000** and it started up without error.
